src/AppBundle/Controller/AdminScoreController.php

```diff
@@ -177,6 +177,8 @@ public function resultAction(Request $request)
             $key = $row['source'];
             $value = $row['value'];
 
+            $value = str_replace("\x00", "", $value);
```

Comment:
Let's add a comment with the task number so there's some context for why we're doing this.